ui/web/index.html

```diff
@@ -3,10 +3,307 @@
+                        sheet: $("#sheetName").val(),
+                        first_row: +$("#firstRow").val(),
+                        last_row: +$("#lastRow").val(),
+                        rows_to_skip: $("#rowSkip").val(),
+                        columns_mapping: {
+                            uid: $("#uid").val(),
+                            name: $("#nam").val(),
+                            blockers: $("#blockers").val(),
+                            min_estimate: $("#minEst").val(),
+                            normal_estimate: $("#norEst").val(),
+                            max_estimate: $("#maxEst").val()
+                        }
+                    };
+                    alert("Response received: " + JSON.stringify(data));
+                    requestBody.file = data.tempFileName;
+                    $.ajax({
+                        type: 'POST',
+                        url: 'http://localhost:5000/api/excelWorkbook/tasks',
+                        contentType: 'application/json',
+                        data: JSON.stringify(requestBody),
+                        success: function (data) {
+                            alert("Response received: " + JSON.stringify(data));                           
+                            applicationContext.tasks = data.tasks;    
+                            var selectList = document.createElement("select");
+                            selectList.setAttribute("id", "tasks");
+                            selectList.setAttribute("name", "tasks");
+                            choiceGraphic.appendChild(selectList);
+                            for (var i = 0; i < applicationContext.tasks.length; i++) {
+                                var option = document.createElement("option");
+                                option.setAttribute("value", i);
+                                option.text = applicationContext.tasks[i].uid;
+                                selectList.appendChild(option);
+                            }
+                        }    
+                    });    
+                }
+            }); 
+        }
+
+        function printTaskGraphic() {
+            var taskToPrintIndex = $("#tasks").val(); 
+            var taskToPrint = applicationContext.tasks[taskToPrintIndex];
+            var inquiryPoints = {
+              "numberOfPoints": 50, 
+               "tasks": [taskToPrint]
+            };
+
+            $.ajax({
+                type: 'POST',
+                url: 'http://localhost:5000/api/tasks/normalDistribution',
+                contentType: 'application/json',
+                data: JSON.stringify(inquiryPoints),
+                success: function drawTaskGraphic(data) {
+                    //alert("Response received: " + JSON.stringify(data));
+                    var tasksPlot = {};                    
+                    tasksPlot.taskNormalDistributions = data.taskNormalDistributions;
+                    var tasksObj = {};
```

Comment:
The name `tasksObj` does not tell a lot about this variable, let's name it something like: `taskNormalDistribution`.

Also it would be better to initialise the variable in same place with its declaration:

```
var taskNormalDistribution = data.taskNormalDistributions[0];
```

Reply:
The variable tasksObj is renamed to taskNormalDistribution